apis/management.cattle.io/v3/authn_types.go

```diff
@@ -25,6 +25,7 @@ type Token struct {
 	Expired         bool              `json:"expired"`
 	ExpiresAt       string            `json:"expiresAt"`
 	Current         bool              `json:"current"`
+	Enabled         *bool             `json:"enabled,omitempty" norman:"default=true"`
```

Comment:
@maxkorp - make sure we have a separate issue to track the fact that you can enable/disable tokens. this is a small enhancment in and of itself and should be tested explicitly. also, open a UI issue so that they can add support to disabling tokens to the UI